charts/caddy-ingress-controller/values.yaml

```
@@ -20,10 +20,10 @@ ingressController:
   rbac:
     create: true
 
-  # If setting autotls the following email value must be set
-  # to an email address that you manage
-  autotls: false
-  email: ""
+  config:
+    acme-ca: ""
```

I renamed `acme-ca` to `acmeCA` and forgot this one.
```suggestion
    acmeCA: ""
```